04/app.js

```diff
@@ -1,9 +1,22 @@
 const user = {
-    firstName: 'Adam',
-    lastName: 'Nowak',
-    born: {
-        day: '14',
-        month: '04',
-        year: '1985'
-    }
-}
\ No newline at end of file
+	firstName: 'Adam',
+	lastName: 'Nowak',
+	born: {
+		day: '3',
+		month: '5',
+		year: '1985',
+	},
+}
+const birthdayShow = function () {
+	const dateToday = new Date()
+	const dayToday = dateToday.getDate()
+	const monthToday = dateToday.getMonth() + 1
+    
+	if (dayToday == 3 && monthToday == 5 ) {
```

Comment:
Powinieneś wykorzystać dane z obiektu `user` tj. `if (dayToday == user.born.day && monthToday == user.born.month ) {` dodatkowo uprościłeś sobie bo miesiąc zapisujesz bez zera, a często to zero występuję - postaraj się tak zmodyfikować kod, aby z zerem przy miesiącu również zadziałało :)